commander/commander.go

```diff
@@ -233,7 +233,7 @@ func getClient(
 	}
 	if cfg.Bootstrap.BootstrapNodeURL != nil {
```

Comment:
I think If we already have chain state in db, we dont need to check with the remote mode. Just ignore the URL in the config and initialize from the saved state.